src/Orckestra.Composer.Product/ViewModels/ProductViewModel.cs

```diff
@@ -1,11 +1,20 @@
-﻿using Orckestra.Composer.ViewModels;
+﻿using Orckestra.Composer.Enums;
+using Orckestra.Composer.ViewModels;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace Orckestra.Composer.Product.ViewModels
 {
     public sealed class ProductViewModel : BaseProductViewModel
     {
+        //Not a lookup in this case.
```

Comment:
let's modify comment to make it more informative.
For example:
In this case the output from Overture contains lookup value, so get display name, need to use Lookup attribute